backend/open_webui/retrieval/reindex_utils.py

```
@@ -0,0 +1,143 @@
+                    return result.model_dump() if result else None
+                except Exception as retry_e:
+                    log.error(f"Query failed even after re-indexing: {retry_e}")
+                    return None
+            else:
+                log.error(f"Re-indexing failed for {file_id}")
+                return None
+        except ImportError:
+            # If we can't import the main app, skip re-indexing
+            log.error(
+                "Cannot import main app for re-indexing, skipping reindex attempt"
+            )
```

Ruff (0.14.3)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)